src/processing/script/tools/acorn/index.js

```diff
@@ -46,3 +48,30 @@ import "./tokenize"
 export function parse(input, options) {
   return new Parser(options, input).parse()
 }
+
+// This function tries to parse a single expression at a given
+// offset in a string. Useful for parsing mixed-language formats
+// that embed JavaScript expressions.
+
+export function parseExpressionAt(input, pos, options) {
+  let p = new Parser(options, input, pos)
+  p.nextToken()
+  return p.parseExpression()
+}
+
+// Acorn is organized as a tokenizer and a recursive-descent parser.
+// The `tokenizer` export provides an interface to the tokenizer.
+
+export function tokenizer(input, options) {
+  return new Parser(options, input)
+}
+
+// This is a terrible kludge to support the existing, pre-ES6
+// interface where the loose parser module retroactively adds exports
+// to this module.
+export let parse_dammit, LooseParser, pluginsLoose
+export function addLooseExports(parse, Parser, plugins) {
+  parse_dammit = parse
+  LooseParser = Parser
+  pluginsLoose = plugins
+}
```

Comment:
It's nice to get rid from everything that we don't need to not grow up our client lib size. I guess we can try to bundle acorn with something like https://github.com/rollup/rollup. Rollup analyzes AST and remove methods and code paths that not used.